### PR TITLE
Added test to exersize queue rebalancing while actively streaming

### PR DESCRIPTION
**Pull request by jason-bragg**
It's a continuation of #4272. GitHub is angry with me and is not letting me force push (this force is not with me) 
This test is a follow-up on stream rebalancing issues raised by @ilyalukyanov in Stream resubscription issues #3484.

### CODE REVIEW

**Comment by jason-bragg** [comment]
test failures:
UnitTests.TimerTests.ReminderTests_TableGrain.Rem_Grain_1J_MultiGrainMultiReminders
Tester.OrleansLoggingTests.OrleansLoggingCanConfigurePerCategoryServeriyOverrides
Tester.OrleansLoggingTests.Logger_BulkMessageLimit_SameFinalLogCode
Tester.OrleansLoggingTests.Logger_BulkMessageLimit_Excludes_100000
Tester.OrleansLoggingTests.MicrosoftExtensionsLogging_Messagebulking_ShouldWork
Tester.OrleansLoggingTests.MicrosoftExtensionsLogging_LoggingFilter_CanAlsoConfigurePerCategoryLogLevel
Tester.OrleansLoggingTests.Logger_BulkMessageLimit_DifferentLoggers
Tester.OrleansLoggingTests.Logger_BulkMessageLimit_DifferentFinalLogCode
Tester.OrleansLoggingTests.Logger_BulkMessageLimit_DebugNotFiltered
ServiceBus.Tests.StreamingTests.EHPersistentStreamQueueRebalancingTests.RebalanceQueuesWhileStreamingTest

Some failures definitely look related to this PR, but volume makes entire test run suspect.  Rerunning.

**Comment by jason-bragg** [comment]
@dotnet-bot test functional please

**Comment by xiazen** [comment]
@dotnet-bot test functional please

**Comment by xiazen** [comment]
No overload for method 'AddEventHubStreams' takes 1 arguments [D:\j\workspace\functional_pr---83d31c72\test\ServiceBus.Tests\ServiceBus.Tests.csproj]

build error. I think you need to rebase with master

File: test/Tester/StreamingTests/PersistentStreamQueueRebalancingTestRunner.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        {
+            var streamGuid = Guid.NewGuid();
+
+            var producer = this.grainFactory.GetGrain<ISampleStreaming_ProducerGrain>(Guid.NewGuid());
+            await producer.BecomeProducer(streamGuid, streamGuid.ToString(), this.streamProviderName);
+
+            var consumer = this.grainFactory.GetGrain<ISampleStreaming_ConsumerGrain>(Guid.NewGuid());
+            await consumer.BecomeConsumer(streamGuid, streamGuid.ToString(), this.streamProviderName);
+
+            IDevelopmentLeaseProviderGrain leaseProviderGrain = InMemoryLeaseProvider.GetLeaseProviderGrain(this.grainFactory);
```
Comment (xiazen) [inline test/Tester/StreamingTests/PersistentStreamQueueRebalancingTestRunner.cs]: I wonder do we need to spin up more consumer to test this. Maybe out of luck, the queue which is feeding this consumer never moved to another silo.

**Comment by xiazen** [comment]
do we still want to merge this PR? if yes please update it. If no, consider close it and keep it under source control, in case we want to bring it back .

**Comment by jason-bragg** [comment]
Closing.  Need a better approach.  This is too unstable.

**Pull request closed without merging**